app/views/inventories/show.html.erb

```diff
@@ -1 +1,14 @@
-<h1> This page would be for a specific inventory </h2>
+<div class="container mt-3 d-flex flex-column align-items-center">
+  <% if @inventory.user == current_user %>
+  <h1><%= @inventory.name %></h1>
+
+  <div class="d-flex justify-content-between w-50 flex-wrap">
+    <%= link_to "Add Food", new_food_path, class: "btn btn-primary m-1" if @inventory.user == current_user %>
```

Comment:
Great job implementing the view, but I think it's better if the link redirects to the form of the relationship between inventories and foods. That would be the `new_inventory_inventory_food(@inventory.id)`